teacher.rb

```diff
@@ -0,0 +1,12 @@
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(specialization, age, name)
```

Comment:
- [ ] Kindly make sure to initialize parent_permission with the given data in the teacher class, also make sure to add `unknown` value to name.
Example:
`(name = 'unknown', parent_permission: true)`